plugins/gnupgplugin/gpgme_p.h

```diff
@@ -294,8 +294,15 @@ struct GPGmeKey {
         QStringList emails;
         gpgme_user_id_t uid = key->uids;
         while (uid) {
-            if (uid->email) {
-                emails << uid->email;
+            if (uid->email && uid->email[0]) {
+                // email can have the form <foo@example.org> or foo@example.org
+                if (uid->email[0] != '<') {
+                    emails << uid->email;
+                } else {
+                    QString email(uid->email + 1);
+                    email.chop(1);
```

Comment:
Just to be a bit overly-defensive, could wrap this email.chop(1) in an if (email.endsWith('>')) check.